frontend/public/components/factory/table-filters.ts

```diff
@@ -19,8 +19,17 @@ import { alertDescription } from '../monitoring/utils';
 import { Target } from '../monitoring/types';
 import { requesterFilter } from '@console/shared/src/components/namespace';
 
-export const fuzzyCaseInsensitive = (a: string, b: string): boolean =>
-  fuzzy(_.toLower(a), _.toLower(b));
+export const fuzzyCaseInsensitive = (a: string, b: string): boolean => {
```

Comment:
Should we keep this function as-is, and add another filter function for the new behavior?

Reply:
I agree, lets keep this one as is and rather add additional filter function.